lib/rucaptcha/controller_helpers.rb

```diff
@@ -73,5 +75,16 @@ def add_rucaptcha_validation_error
       end
       false
     end
+
+    def notify_when_session_invalid
+        puts "
+
+        Warning from RuCaptcha:
+        Your session is not yet loaded, the session.id or session['session_id'] is blank.
+        Please keep session available, otherwise you can't use RuCaptcha rightly.
+        More info please check https://github.com/huacnlee/rucaptcha/blob/master/README.md#Traps
```

Comment:
Just give a simple warning message.

```
WARNING! session.id is blank, RuCaptcha can not work, please keep session available.
More detail about this: https://github.com/huacnlee/rucaptcha/pull/66
```

Reply:
Thanks for your nice comments. I will do it as soon as possible.